airflow-core/src/airflow/ui/src/i18n/locales/tr/common.json

```diff
@@ -0,0 +1,122 @@
+{
+  "admin": {
+    "Config": "Konfigürasyon",
+    "Connections": "Bağlantılar",
+    "Plugins": "Eklentiler",
+    "Pools": "Havuzlar",
+    "Providers": "Sağlayıcılar",
+    "Variables": "Değişkenler"
+  },
+  "assetEvent_one": "Varlık Olayı",
+  "assetEvent_other": "Varlık Olayları",
+  "browse": {
+    "auditLog": "Denetim Günlüğü",
+    "xcoms": "XComs"
+  },
+  "dag_one": "dag",
```

Comment:
In Turkish, are borrowed terms uncapitalized? (Assuming that the original term is Dag)